.github/workflows/install-pnpm/action.yaml

```diff
@@ -8,12 +8,8 @@ runs:
         version: 9
         run_install: false
 
-    - name: 🤞 Install Node.js
+    - name: 🤞 Use Node.js
       uses: actions/setup-node@v4
       with:
         cache: pnpm
         node-version: 21
-
-    - name: ⚙️ Install dependencies
```

Comment:
Why do you remove dependency install from this action. If you need to install pnpm separately - create new action "install pnpm" and reuse it here